File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/python/IvanHernandezHe.py
def FizzBuzz(limit):
    for i in range(1,limit+1):
        print("FizzBuzz" if ((i%3==0) & (i%5==0)) else "Fizz" if (i%3==0) else "Buzz" if (i%5==0) else i)
        
FizzBuzz(100)

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/typescript/IvanHernandezHe.ts
FizzBuzz(100)

function FizzBuzz (limit)
{
    for (let i = 1; i <= limit; i++)
    {
        console.log((( i % 3 == 0) && ( i % 5 == 0)) ? "Fizzbuzz" : ( i % 3 == 0) ? "Fizz" : ( i % 5 == 0) ? "Buzz" : i)
    }
}